app/Http/controllers/postController.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require('mongoose')
+const Post = require('../../models/Post.js')
+
+module.exports = () => {
+  var post = {}
+
+  post.index = (req, res) => {
+    Post.find(function(errors, posts) {
+      if (errors) {
+        log('Errors occur: ', errors)
+        res.send(errors) // status 500
+      } else {
+        res.render('../../views/posts/index', {posts: posts})
```

Comment:
Linting { post: posts } (destructing object, need space before and after brace in object )